This is to fix the following test failure:

  [ext/standard/tests/streams/stream_context_tcp_nodelay_server.phpt]
  ========DIFF========
  001+
  001- server-delay:conn-nodelay
  ========DONE========

The test runs a server in a separate php process which fails with the
following error:

  Fatal error: Uncaught Error: Call to undefined function socket_get_option() in ext/openssl/tests/ServerClientTestCase.inc(125) : eval()'d code:15

The reason is the process is not properly configured and doesn't use the
'sockets' exetnsion.

--
From 5a4083181b970411d6f3b1c1e3bdd199d5cd1a5e Mon Sep 17 00:00:00 2001
From: Jakub Holubansky <jakub.holubansky@team.bumble.com>
Date: Fri, 31 Mar 2023 17:20:35 +0200
Subject: [PATCH] ext/openssl: pass ini options to extra processes in tests

---
 ext/openssl/tests/ServerClientTestCase.inc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/ext/openssl/tests/ServerClientTestCase.inc b/ext/openssl/tests/ServerClientTestCase.inc
index 753366df6f4..1b140b63f6b 100644
--- a/ext/openssl/tests/ServerClientTestCase.inc
+++ b/ext/openssl/tests/ServerClientTestCase.inc
@@ -72,8 +72,9 @@ class ServerClientTestCase
             );
         } else {
             $cmd = sprintf(
-                '%s "%s" %s %s',
+                '%s %s "%s" %s %s',
                 PHP_BINARY,
+                getenv('TEST_PHP_EXTRA_ARGS'),
                 __FILE__,
                 WORKER_ARGV_VALUE,
                 $worker
-- 
2.48.1

